### ISSUE TITLE
improve feedback to use on whether settings are updated or not

### ISSUE THREAD

**Issue by jmcclelland** [progressivetech/net.ourpowerbase.sumfields#32]
Now that we have the ability to update data via a cron job (instead of triggers), we have a 
communications problem.

If the status is "scheduled" - it could mean that the settings are not saved. Or, it could mean that the settings are saved, but we are always in a scheduled status since we update on every cron job.

I think the proper solution is to use two different variables - one to keep track of whether the settings are saved and another to keep track of whether the data needs to be updated.

Also, would be helpful to include the fact that we are using the cron method for updating data up top, where we would list that all the sql triggers are not enabled.

**Comment by VangelisP** [comment]
This is true, right now it's not exactly clear what's happening if you update data via cron, or better .. it's confusing.

I'll work a little bit on this part and the admin part as well

**Comment by jmcclelland** [comment]
Great - thanks!
